tests/benchdnn/graph/flex_rewrite.cpp

```
@@ -1329,6 +1330,57 @@ void flex_rewrite_t::dt_rewrite(deserialized_graph_t &dgraph) {
     }
 }
 
+void flex_rewrite_t::op_kind_rewrite(deserialized_graph_t &dgraph) {
+    // Step 1: check the op kind in the map and whether the given ids are in
+    // the graph.
+    for (const auto &v : op_kind_map_) {
+        if (v.second == "default") return;
+
+        auto target_kind = opstr2kind(v.second);
+        if (target_kind == dnnl::graph::op::kind::LastSymbol) return;
```

Seems like an incorrect `op_kind` will be silently ignored which is not good. A message and an error returning should be here.

---

Thanks for reminding. I added more error message here.